logistechMOTHAFUCKA/logistechMOTHAFUCKA.ino

```diff
@@ -169,10 +113,34 @@ void loop() {
       }
       // Free resources
       http.end();
+      num_codes++;
+      Serial.print("New unique code: ");
+      Serial.println(new_code, HEX);
+      Serial.print("Total unique codes: ");
+      Serial.println(num_codes);
     }
-    else {
-      Serial.println("WiFi Disconnected");
+    else if(!is_unique){
+      int httpResponseCode = http.POST("{\"longitude\": 48.2082, \"latitude\": 16.3738, \"productItemId\": \"" + String (new_code) + "\"}");
+      
+      if (httpResponseCode>0) {
+        Serial.print("HTTP Response code: ");
+        Serial.println(httpResponseCode);
+        String payload = http.getString();
+        Serial.println(payload);
+      }
+      else {
+        Serial.print("Error code: ");
+        Serial.println(httpResponseCode);
+      }
```

Comment:
@princhcanal  you may simplify this part too. like the validation of the response we can put it outside` if(is_unique)` and `else if(! if(is_unique))` so it wont repetitive

Reply:
duly noted as well 📓